js/ui/overlay/ui.overlay.js

```diff
@@ -479,6 +485,22 @@ const Overlay = Widget.inherit({
         this._refresh();
     },
 
+    _renderWrapperAttributes(wrapperAttr = this.option('wrapperAttr') || {}) {
+        const attributes = extend({}, wrapperAttr);
+        const classNames = attributes.class;
+        delete attributes.class;
+        if(this._oldAttributes) {
+            this._wrapper().removeAttr(Object.keys(this._oldAttributes).join(' '));
```

Comment:
Seems like it's good idea to get rid of the condition and use nullish coalescing operator instead

Reply:
done